editor.planx.uk/src/@planx/components/Pay/model.ts

```diff
@@ -63,6 +67,11 @@ export const createPayload = (
   reference,
   description: "New application",
   return_url: getReturnURL(reference),
+  metadata: {
```

Comment:
I think this might be redundant if we're just appending it in the API anyway - it might be simpler just to handle this once server-side.

Reply:
The change to `middleware.ts` actually only impacts invite-to-pay payment requests ! So this change to the frontend model is the _only_ place it's handled for all "regular" payments (confusingly so! there's more duplication and separation in the invite to pay work than I remembered - one to revisit someday!)

Reply:
Ah good catch - thanks for clarifying that. Yep, one to tidy up at some point!